interfaces/apparmor/backend.go

```diff
@@ -512,7 +526,17 @@ func addContent(securityTag string, snapInfo *snap.Info, opts interfaces.Confine
 
 func reloadProfiles(profiles []string, profileDir, cacheDir string) error {
 	for _, profile := range profiles {
-		err := loadProfile(filepath.Join(profileDir, profile), cacheDir)
+		err := loadProfile(filepath.Join(profileDir, profile), cacheDir, 0)
+		if err != nil {
+			return fmt.Errorf("cannot load apparmor profile %q: %s", profile, err)
+		}
+	}
+	return nil
+}
+
+func reloadChangedProfiles(profiles []string, profileDir, cacheDir string) error {
```

Comment:
Curious why you didn't change the arg list of reloadProfiles instead of adding this. Was that to make the patch simple and easy to back out later when apparmor_parser uses a hash instead of mtimes?

Reply:
It was just a hunch, I didn't want to add flags at this level and instead I wanted to make it evident from the function name itself, that something different is going on.